### DISCUSSION TITLE
Why are my backpacks invisible?

### DISCUSSION THREAD

**Discussion by harouo** [P3pp3rF1y/SophisticatedBackpacks#1169] [Q&A]
the backpacks appear normally on others and my back from third person perspective but are invisible in the inventory, all other functions works fine its just the backpack thats visually gone in the inventory, this is on a fabric server using 0.16.5 on 1.20.1
![image](https://github.com/user-attachments/assets/a46c970a-7725-4a8f-8c01-2fc3fee4f277)
![image](https://github.com/user-attachments/assets/03ad03ba-09bf-4475-97d0-81720de0cb44)
the list of mods on this server:
![image](https://github.com/user-attachments/assets/e636fb26-92e1-4363-bbb4-a153b22edffb)

**Comment by P3pp3rF1y** [comment]
you probably wanted to report this to the unofficial fabric port of SB I can't help you with that one
